common/djangoapps/student/models.py

```diff
@@ -1569,7 +1569,11 @@ def unenroll_by_email(cls, email, course_id):
         """
         try:
             user = User.objects.get(email=email)
-            return cls.unenroll(user, course_id)
+            record = cls.objects.get(user=user, course_id=course_id)
+            if record.refundable():
+                return record.update_enrollment(mode=CourseMode.AUDIT, is_active=False, skip_refund=False)
```

Comment:
I am concerned about adding a change to the users mode on an unenroll.  This seems to have been left out deliberately.  Why would we change a mode when we are not refunding the user?